app/assets/stylesheets/_pages.scss

```diff
@@ -1,3 +1,8 @@
+html {
+  overflow: -moz-scrollbars-vertical;
+  overflow-x: hidden;
+  overflow-y: hidden;
+}
```

Comment:
I'll have to rethink the vertical scrollbar, it could make sense to have infinite scrolling rather than pagination 🤷‍♀️